README.md

```diff
@@ -46,16 +47,17 @@ Look at `package.json` for details and more scripts.
 
 ### Publishing the package on npm
 
-1. Ensure you are on a `main` branch
-2. Simply run:
+1. `git checkout main` - to ensure you are on a `main` branch
+2. `yarn install` - to install fresh dependencies
```

Comment:
I've added `yarn` command to `prepublishOnly` to make sure the dependencies are really up to date.

I'm adding it to docs since:
1. it's possible that when using `yarn` the yarn.lock file will change in which case you might want to commit it beforehand
2. It doesn't hurt performance, since the next `yarn install` will verify all is up to date and finish early. (this is not the case with build, which will remove dist folder and start again.)